### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by jorilallo** [kippt/kippt-chrome#2]
I need to get this done sometime soon with this: http://craig.is/killing/mice

What would be the ideal keyboard combination?

**Comment by thierrylemoulec** [comment]
Something like `⌘ + k` or `⌘ + ⇧ + k` would be nice and doesn't seem to override one of the existing chrome shortcuts http://thry.me/5bew

**Comment by jorilallo** [comment]
I had problems passing the data to the extension, but might work around the issues later.

**Comment by mrmurphy** [comment]
I just wanted to throw in my vote that keyboard shortcuts are essential. Kippt will really solve some of my workflow problems if you get this working.

**Comment by d2s** [comment]
Yeah, I might have suggested this already but shortcuts would be awesome.

Thought, not sure of what would be the "best" key combination since some people always have conflicting keys. Whatever it is by default, it should be configurable.

**Comment by mwaterfall** [comment]
+1 for customisable keyboard shortcuts like Readability! I use Kippt so much that I'd happily assign the backtick key to bring up the Kippt prompt.
